### PR TITLE
[202012][sonic_sfp]: prefer Config DB port index mapping over platform.json/port_config.ini

### PR DESCRIPTION
**Pull request by nazariig**
**Approach:**
1. Always use Config DB to get port mapping when it's available
2. When port mapping is missing in Config DB, use either `platform.json` or `port_config.ini`

Signed-off-by: Nazarii Hnydyn <nazariig@nvidia.com>

### CODE REVIEW

File: sonic_platform_base/sonic_sfp/sfputilhelper.py [hunk 1]
```diff
@@ -45,6 +47,22 @@ class SfpUtilHelper(object):
     def __init__(self):
         pass
 
+    def get_port_mapping(self):
```
Comment (nazariig) [inline sonic_platform_base/sonic_sfp/sfputilhelper.py]: - [x] **FIXME:** Multi ASIC is not taken into consideration in this approach

File: sonic_platform_base/sonic_sfp/sfputilhelper.py [hunk 2]
```diff
@@ -45,6 +47,19 @@ class SfpUtilHelper(object):
...
+    def get_port_mapping(self, asic_inst):
+        if multi_asic.is_multi_asic():
+            ports_data = multi_asic.get_port_table_for_asic("{}{}".format(multi_asic.ASIC_NAME_PREFIX, asic_inst))
+        else:
+            ports_data = multi_asic.get_port_table_for_asic(multi_asic.DEFAULT_NAMESPACE)
+
+        ports = ast.literal_eval(json.dumps(ports_data))
+        for port in ports.keys():
+            if "index" not in ports[port]:
+                return None
```
Comment (Junchao-Mellanox) [inline sonic_platform_base/sonic_sfp/sfputilhelper.py:65]: Log something here?

**Comment by Junchao-Mellanox** [comment]
LGTM except a minor comment.

File: sonic_platform_base/sonic_sfp/sfputilhelper.py (hunk 2 above)
Comment (nazariig) [reply sonic_platform_base/sonic_sfp/sfputilhelper.py:65]: @Junchao-Mellanox done

**Review by Junchao-Mellanox — APPROVED** [review]

**Pull request closed without merging**